src/YoutubeAtom/YoutubeMeta.tsx

```diff
@@ -77,6 +78,31 @@ const Icon = ({ pillar }: { pillar: Pillar }) => (
     </span>
 );
 
+const svgStyle = css`
+    left: 35%;
+    top: 1%;
```

Comment:
Does this need to be % when you've set the height and width of the button in px? I can't see where these styles are used?